rust/agama-server/src/network/web.rs

```diff
@@ -101,10 +101,10 @@ pub async fn network_service<T: Adapter + Send + Sync + 'static>(
                 .put(update_connection)
                 .get(connection),
         )
-        .route("/connections/:id/connect", get(connect))
-        .route("/connections/:id/disconnect", get(disconnect))
+        .route("/connections/:id/connect", patch(connect))
+        .route("/connections/:id/disconnect", patch(disconnect))
         .route("/devices", get(devices))
-        .route("/system/apply", put(apply))
+        .route("/system/apply", post(apply))
```

Comment:
what is rationale for this change? for me the main difference between post and put is idempotency. So when I call apply twice will it do two actions and only the first apply do something?

Reply:
If I understood what @imobachgs told me last week, the rationale here is that we're using POST for these kind of actions, like it's the case of _probing_.